app/views/orders/new.html.haml

```diff
@@ -0,0 +1,44 @@
+.row
```

Comment:
Will take a look how it looks as a UI. Can't understand how you can make an order